docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -34,9 +34,35 @@ Failed installing package [apm] due to error: [ResponseError: mapper_parsing_exc
 		mapper_parsing_exception: Field [jvm.memory.non_heap.pool.committed] attempted to shadow a time_series_metric]
 ----
 
-// Link to fix?
+// Link to fix
 A fix was released in 8.11.2: https://github.com/elastic/kibana/pull/171712[elastic/kibana#171712].
 
+[[apm-empty-metricset-values]]
+*Upgrading APM Server to 8.11+ might break event intake from older APM Java Agents* +
```

Comment:
Should this be above the older KI at the top of the page?

Reply:
🤔  I'm not sure. Is that how we do it for other products?

Reply:
Dunno. It's what we do with APM Server release notes

Reply:
After a quick search of other "known issues" pages, it looks like the content is organized by topic rather than by version number. 🤷‍♀️  So I think we can do our own thing here. I moved the template to the top of the page and added a note to add new known issues for newer versions to the top.